installers/yarn2nix.nix

```diff
@@ -1,30 +1,27 @@
-{ lib, pkgs, nodejs-8_x, python, api, apiVersion, cluster, buildNum, nukeReferences, fetchzip, daedalus, stdenv }:
+{ lib, pkgs, nodejs-8_x, python, backend, nukeReferences, version, fetchzip, stdenv, src }:
 let
   nodejs = nodejs-8_x;
   yarn2nix = import (fetchzip {
     url = "https://github.com/moretea/yarn2nix/archive/v1.0.0.tar.gz";
     sha256 = "02bzr9j83i1064r1r34cn74z7ccb84qb5iaivwdplaykyyydl1k8";
   }) { inherit pkgs nodejs; };
-  networkMap = {
-    mainnet = "mainnet";
-    staging = "testnet";
-    testnet = "testnet";
-  };
-  dotGitExists = builtins.pathExists ./.git;
-  isNix2 = 0 <= builtins.compareVersions builtins.nixVersion "1.12";
-  canUseFetchGit = dotGitExists && isNix2;
 in
 yarn2nix.mkYarnPackage {
-  name = "daedalus-js";
-  src = if canUseFetchGit then builtins.fetchGit ./. else lib.cleanSource ./.;
-  API = api;
-  API_VERSION = apiVersion;
+  name = "daedalus-frontend";
+  inherit src;
+  # Provide direct paths to yarn sources so that
+  # import-from-derivation works in sandboxed build.
+  yarnLock = ../yarn.lock;
+  packageJSON = ../package.json;
+
+  API = backend.api;
+  API_VERSION = backend.version;
   CI = "nix";
-  NETWORK = networkMap.${cluster};
-  BUILD_NUMBER = "${toString buildNum}";
+  DAEDALUS_VERSION = "${version}";
+  # BUILD_NUMBER = toString buildNum;
```

Comment:
Should we just drop this instead of leaving it commented out?

Reply:
Yes - fixed.